README.md

```diff
@@ -1,96 +1,98 @@
+Bloom consists of a client/server architecture using [Next.js](https://nextjs.org) for the frontend applications and [NestJS](https://nestjs.com), [Prisma](https://www.prisma.io/), and [Postgres](https://www.postgresql.org/) on the backend.
 
 ### Structure
 
-Bloom uses a monorepo-style repository containing multiple user-facing applications and backend services. The three main high-level packages are `api`, `sites`, and `shared-helpers`. Additionally, Bloom's UI leverages the in-house npm package `@bloom-housing/ui-components`.
+Bloom uses a monorepo-style repository containing multiple user-facing applications and backend services. The three main high-level packages are `api`, `sites`, and `shared-helpers`. Additionally, Bloom's UI leverages the in-house packages `@bloom-housing/ui-seeds` and `@bloom-housing/ui-components`.
 
-The `sites` package contains reference implementations for the two user-facing applications in the system:
+The `sites` folder contains reference implementations for both the public and partner applications:
 
 ---
 
-- `sites/public` is the applicant-facing site available to the general public. It provides the ability to browse available listings and to apply for listings either using the Common Application (which we build and maintain) or an external link to a third-party online or paper application.
+- `sites/public` is the applicant-facing site available to the general public. It provides the ability to browse and apply for available listings either using the Common Application (which we build and maintain) or an external link to a third-party online or paper application.
 - Visit [sites/public/README](https://github.com/bloom-housing/bloom/blob/main/sites/public/README.md) for more details.
 
 - `sites/partners` is the site designed for housing developers, property managers, and city/county (jurisdiction) employees. For application management, it offers the ability to view, edit, and export applications for listings and other administrative tasks. For listing management, it offers the ability to create, edit, and publish listings. A login is required to use the Partners Portal.
 - Visit [sites/partners/README](https://github.com/bloom-housing/bloom/blob/main/sites/partners/README.md) for more details.
 
-In some cases the sites diverge slightly to accomodate jurisdictional customizations. The [housingbayarea Bloom fork](https://github.com/housingbayarea/bloom) is a fork of Bloom core for Bay Area jurisdictions which is loosely customized for that location. In this fork, our jurisdictions are each a separate branch.
+In some cases for the sites folder, production deployments in jurisdictions will diverge to accommodate customizations. For example, [housingbayarea](https://github.com/housingbayarea/bloom) (HBA) is a fork of Bloom core that supports multiple Bay Area jurisdictions, which is lightly customized for each jurisdiction. In HBA, our jurisdictions are each a separate branch.
```

Comment:
I didn't realize that we reference HBA in core's README. I wonder if we should remove this section all together, especially since we are hoping to one day not fork for new jurisdictions.

If we think it's still useful information, can you remove the line about "separate branch" being that that is no longer valid.

Reply:
oh good call! i think it makes sense to remove hba references